app/main.py

```diff
@@ -1 +1,11 @@
-# write your code here
+def copy_file(command: str) -> None:
+    if len(command.split()) == 3:
+        _command, file, new_file = command.split()
+        if file == new_file:
+            return
+        if _command.lower() == "cp":
```

Comment:
Ensure that the `cp` input command is correct. The checklist specifies that the `cp` input command should be checked for correctness.